docs/release_notes.md

```diff
@@ -0,0 +1,78 @@
+# Release Notes
+
+This document tracks the major changes in each package release of TurnkeyML.
+
+We are tracking two types of major changes:
+ - New features that enhance the user and developer experience
+ - Breaking changes to the CLI or public APIs
+
+If you are creating the release notes for a new version, please see the [template](#template-version-majorminorpatch). Release notes should capture all of the significant changes since the last numbered package release.
+
+# Version 1.0.0
+
+This version focuses on cleaning up technical debts. It removes cumbersome requirements and unused features, while also clarifying some naming schemes.
+
+## Improvements
+
+### CLI Users
+
+Improvements to the information in `turnkey_stats.yaml` and report CSVs:
+ 
+ - Now reports all model labels. Including, but not limited to, the model's OSS license.
+ - `build_status` and `benchmark_status` now accurately report the status of their respective toolchain phases.
+     - Previously, `benchmark_status` was a superset of the status of both build and benchmark.
+
+### API Developers
+
+ - Build success has been conceptually reworked for Stages/Sequences such that the `SetSuccess` Stage is no longer required at the end of every Sequence.
+   - Previously, `build_model()` would only return a `State` object if the `state.build_status == successful_build`, which in turn had to be manually set in a Stage.
+   - Now, if a Sequence finishes then the underlying toolflow will automatically set `state.build_status = successful_build` on your behalf.
+
+## Breaking Changes
+
+### CLI Users
+
+None.
+
+### API Developers
+
+ - The `benchmark_model()` API has been removed as there were no known users / use cases. Anyone who wants to run standalone benchmarking can still instantiate any `BaseRT` child class and call `BaseRT.benchmark()`.
+ - The APIs for saving and loading labels `.txt` files in the cache have been removed since no code was using those APIs. Labels are now saved into `turnkey_stats.yaml` instead.
+ - The `quantization_samples` argument to the `build_model()` API has been removed.
+ - The naming scheme of the members of `Stats` has been adjusted for consistency. It used to refer to both builds and benchmarks as "builds", whereas now it uses "evaluations" as a superset of the two.
+   - `Stats.add_build_stat()` is now `Stats.save_model_eval_stat()`.
+   - `Stats.add_build_sub_stat()` is now `Stats.save_model_eval_sub_stat()`.
+   - `Stats.stat_id` is now `Stats.evaluation_id`.
+   - The `builds` section of the stats/reports is now `evaluations`.
+   - `Stats.save_stat()` is now `Stats.save_model_stat()`.
+   - `Stats.build_stats` is now `Stats.evaluation_stats`.
+ - The `SetSuccess` build stage has been removed because build success has been reworked (see improvements).
+ - The `logged_subprocess()` API has been moved from the `common.build` module to the `run.plugin_helpers` module.
```

Comment:
Should we also add those to the "Describe this release" field when creating the actual release?
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

Reply:
We aren't managing releases in any formal way (according to GitHub) right now. But something to look into in the future!